### PR TITLE
Remove verify_*, enable param docs linting

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -1,4 +1 @@
 ---
```
Comment (ekohl) [inline .sync.yml]: Remove the entire file?

Comment (domcleal) [inline .sync.yml]: Sorry, I forgot to add the new lint pattern here.  Fixed!

**Comment by ekohl** [comment]
:+1: on green tests.

**Comment by mmoll** [comment]
thanks @domcleal!

**Pull request merged**